numpy/core/src/common/wrapper.h

```diff
@@ -0,0 +1,13 @@
+#ifndef _COMMON_WRAPPER_H
```

Comment:
This file name is pretty cryptic - I'd hope there's a collection of similar functions already we can put this with. `numpyos.c` has `strtold`, which is pretty similar - perhaps it should go there.

Reply:
adopted